pom.xml

```diff
@@ -30,7 +30,7 @@
 	</parent>
 	<groupId>ca.on.oicr.icgc.argo</groupId>
 	<artifactId>program-service</artifactId>
-	<version>1.16.0</version>
+	<version>1.17.0-SNAPSHOT</version>
```

Comment:
can you remove SNAPSHOT from the new version?

Reply:
Okay. will be doing it

Reply:
modified the program service version. Please review